### PR TITLE
Docs: add MDL Sass guide

### PR DESCRIPTION
**Pull request by addyosmani**
Review: @surma @sgomes 

Fyi:  @garyterry who was trying to do this with MDL over the weekend

Notes: ideally, this work would be done in some `third_party` directory within WSK. This however, would have introduced several more steps so I decided to do all the work in the `app/styles` directory. Imo, this reduces overhead for getting setup.

### CODE REVIEW

**Comment by surma** [comment]
Awesome, @addyosmani. Thanks for whipping this up. LGTM :) 

I really dislike the huge JS import block, but I don’t think there’s a way around that right now.

File: docs/mdl-sass.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+For this example, we're going to change out `palette-indigo-500` for `palette-purple-500` and `palette-indigo-700` for `palette-deep-purple-700`.
+
+```
+$color-primary: $palette-purple-500 !default;
+$color-primary-dark: $palette-deep-purple-700 !default;
+$color-accent: $palette-pink-A200 !default;
+```
+
+Make the above change and hit save. The page should refresh, now showing you your customised theme in action. 
+
```
Comment (sgomes) [inline docs/mdl-sass.md]: You need to set two other colours: `$color-primary-contrast` and `$color-accent-contrast`. These are the colours for text that gets rendered on top of a solid block of primary or accent, respectively. You should set them to `$color-dark-contrast` if you've chosen a dark primary/accent, and `$color-light-contrast` if you've chosen a light primary/accent.

**Comment by sgomes** [comment]
LGTM, other than the one comment I made. Thanks for the write-up! :)

**Comment by addyosmani** [comment]
Thanks for the feedback! Incorporated into the end of the doc!

**Comment by garyterry** [comment]
FYI, I'm going through the steps now... I'll have time to document progress over the weekend. Thanks for getting this done so quickly! YOU GUYS ROCK! :+1:

**Comment by garyterry** [comment]
Just got it working on windows at work. No installation issues to report. I'll be pulling in the templates from the MDL library to see if I can get all of it playing nicely together tomorrow... Looking pretty solid so far... excited!

**Comment by addyosmani** [comment]
Fantastic to hear. Thanks a lot for reporting back! :)

**Pull request merged**